### PR TITLE
Feature/sf live 2017

### PR DESCRIPTION
**Pull request by fabdsp**
Retour le SF Live Paris 2017.

### CODE REVIEW

File: _posts/2017-04-03-sf-live-2017.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+image:
+  feature: posts/sflive2017/SfLive2017.jpg
+  credit: Fabien De Saint Pern
+  creditlink:
+comments: true
+language: en
+---
+
+In March we attended [Symfony Live Paris 2017](http://paris2017.live.symfony.com/speakers), and it was very interesting.
+Here some special feedbacks about some of our favorite talks.
```
Comment (Oliboy50) [inline _posts/2017-04-03-sf-live-2017.md]: s/Here some/Here are some

moyen sûr mais... 🤔

File: _posts/2017-04-03-sf-live-2017.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+### How to tag and segment the cache
+By tagging every resources by a unique id (`entityName_id` by example), you'll be able to invalidate later the cache linked to it.
+You can simplify and automate this tagging by using event dispatcher and listen all entities creations / modifications. With
+this solution you also centralize your code logic.
+
+Cache tagging also helps you with caching / invalidating many-to-many relationships between content items.
+
+### How to invalidate the cache
+Jérémy showed us a very interesting interface, displaying the entity content, automatically refreshed when the backend needs to be called
+because of a cache invalidation.
```
Comment (Oliboy50) [inline _posts/2017-04-03-sf-live-2017.md]: saut de ligne inutile ?

File: _posts/2017-04-03-sf-live-2017.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+Once a resource is modified and needs to be uncached, your backend needs to call the Varnish to purge the tag linked to
+resources. Once again the [FOS HttpCache bundle](http://foshttpcachebundle.readthedocs.io/en/latest/reference/tag-handler.html) does it very well.
+
+### Advantages / disadvantages
```
Comment (Oliboy50) [inline _posts/2017-04-03-sf-live-2017.md]: s/disadvantages/drawbacks

File: _posts/2017-04-03-sf-live-2017.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+### Resources
+Link to the speaker presentation : https://speakerdeck.com/nicolasgrekas/tout-ce-quun-dev-devrait-savoir-a-propos-dunicode
+
+## Performances optimisation with Php7
+### Speaker: Julien Pauli
+### Description
+
+As usual, it was a real pleasure to listen Julien Pauli speaking about what's under the hood of Php7.
+Everybody knows there is an actual performances gap between version 5 and 7, but this talk gave some clues to understand
+technical reasons of these major improvements.
```
Comment (Oliboy50) [inline _posts/2017-04-03-sf-live-2017.md]: s/to understand technical reasons of these/to understand the technical reasons behind these

**Comment by omansour** [comment]
go merge @fabdsp

**Pull request merged**